src/common/transformations/src/transformations/fp16_compression/mark_subgraphs_to_keep_in_mixed_precision.cpp

```diff
@@ -423,6 +424,35 @@ class PropagateDownDisableSensitivityForQuantized : public pass::MatcherPass {
     }
 };
 
+/* FloorDiv should be kept in fp32 precision for better accuracy
+   when it accepts large values (> fp16_max).
+*/
+class MarkFloorDiv : public pass::MatcherPass {
+public:
+    OPENVINO_RTTI("MarkFloorDiv", "0");
+    MarkFloorDiv() {
+        MATCHER_SCOPE(MarkFloorDiv);
+        // FloorDiv is repesented either as:
+        // Floor(Div(input_1, input_2))
+        // Floor(Mul(input_1, inversed_const_input_2)), if input_2 is constant
+        auto div_mul_pattern = pattern::wrap_type<ov::op::v1::Divide, ov::op::v1::Multiply>();
+        auto floor_pattern = pattern::wrap_type<ov::op::v0::Floor>({div_mul_pattern});
```

Comment:
should we check that 2nd input to Mul is constant according to the comment?
 // Floor(Mul(input_1, inversed_const_input_2)), if input_2 is constant

Reply:
Agreed, i corrected the pattern. No need to check Const values/types, they should be marked in any case.